docs/source/upgrade/example/coin-1.0.0/daml.yaml

```diff
@@ -5,10 +5,10 @@ sdk-version: 0.0.0
 # BEGIN
 name: coin
 version: 1.0.0
-parties: [Alice, Bob]
 dependencies:
   - daml-prim
   - daml-stdlib
 # END
+parties: [Alice, Bob]
```

Comment:
What's the purpose of hiding the parties?... They are mentioned in the following example steps.

Reply:
I’d like to focus on the stuff that is specific or relevant for upgrades. The `parties` field is imho more of a hack because we do not properly allocate parties here (and even if we did, navigator doesn’t pick them up unless you use `daml ledger navigator` which requires restarting, …) so I think it just distracts here.